auth-service/src/main/kotlin/uk/gov/communities/delta/auth/plugins/StatusPages.kt

```diff
@@ -29,7 +29,8 @@ fun Application.configureStatusPages(deltaWebsiteUrl: String, ssoConfig: AzureAD
                             "register-user-form",
                             mapOf(
                                 "deltaUrl" to deltaConfig.deltaWebsiteUrl,
-                                "allErrors" to arrayListOf(arrayListOf(tooManyRequestsErrorMessage, "#")),
+                                "allErrors" to arrayListOf(Pair(tooManyRequestsErrorMessage, "#")),
+                                "errorSummary" to "Rate limit reached",
```

Comment:
Does this still fit the gov guidance to have text other than "There is a problem" here? I agree it's clearer for sure tho

Reply:
I guess not, it does say it should always be "There is a problem". I think it's clearer anyway so I'll leave it